packages/legacy-workbench/src/js/angular/security/controllers.js

```diff
@@ -577,25 +589,37 @@ securityModule.controller('AddUserCtrl', ['$scope', '$http', 'toastr', '$window'
 
         $scope.createUserHttp = function() {
             $scope.loader = true;
-            SecurityService.createUser({
+            const user = new User({
                 username: $scope.user.username,
-                pass: $scope.user.password,
-                appSettings: $scope.user.appSettings,
-                grantedAuthorities: $scope.user.grantedAuthorities,
-            }).then(() => {
-                toastr.success($translate.instant('security.user.created', {name: $scope.user.username}));
-                const timer = $timeout(function() {
+                password: $scope.user.password,
+                appSettings: new AppSettings($scope.user.appSettings),
+                authorities: new AuthorityList($scope.user.grantedAuthorities),
+            });
+            usersService.createUser(user)
+                .then(() => {
+                    toastr.success($translate.instant('security.user.created', {name: $scope.user.username}));
+                    const timer = $timeout(function() {
```

Comment:
This BS was needed just for the sake of showing the toastr. Right?

Reply:
Propably because of the `window.history.back` BS.

I guess it will be cleaned when migrating the view.

There is a nice pause now when the user is created